KarhooUISDK/Screens/AddressScreen/AddressViewController.swift

```diff
@@ -157,12 +157,7 @@ final class AddressViewController: UIViewController, AddressView {
     }
 
     private func configureForAuthenticationMethod() {
-        if Karhoo.configuration.authenticationMethod().guestSettings == nil {
-            buildAddressMapView()
-        } else {
-            setOnMapView.isHidden = true
-            currentLocationView.isHidden = true
-        }
+        buildAddressMapView()
```

Comment:
Is it possible in iOS to check if the application has access to Location service or not? 

If the application is denided to access the location, we need to hide the currentLocationView and setOnMapView (similar to Android)

```
setOnMapView.isHidden = false
currentLocationView.isHidden = false
```